src/sqlfluff/rules/L016.py

```diff
@@ -263,8 +280,9 @@ def generate_fixes_to_coerce(
         elif indent_impulse:  # pragma: no cover
             role = "breakpoint"
         else:
-            raise ValueError("Is this possible?")  # pragma: no cover
-
+            # This can happen, e.g. with a long template line. Treat it as
+            # unfixable.
+            return []
```

Comment:
Is this how we return an unfixable line? I've always used `return LintResult()`. Or is that only for when there is nothing to fix?

Reply:
No, not normally. This code is inside a function `generate_fixes_to_coerce()`, which is called by `eval()`. As you say, `eval()` itself optionally returns a `LintResult`. This particular function returns a list of LintFix objects, or `List[LintFix]` in type-hint-speak.

Reply:
This rule is pretty large and complex, which is why I added more type hints. They provide a bit more guidance about what bits of the code expect from one another.